lexicons/app/bsky/graph/getFollowsOfFollows.json

```diff
@@ -0,0 +1,34 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.graph.getFollowsOfFollows",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Get the accounts followed by the accounts followed by a given actor.",
+      "parameters": {
+        "type": "params",
+        "required": ["actor"],
+        "properties": {
+          "actor": {"type": "string", "format": "at-identifier"},
+          "limit": {"type": "integer", "minimum": 1, "maximum": 100, "default": 50},
+          "cursor": {"type": "string"}
```

Comment:
Note to self: this query will probably be a little perf intensive, so maybe we want to handle pagination differently, or not at all